knative-operator/pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -323,7 +343,7 @@ func (r *ReconcileKnativeServing) installQuickstarts(instance *operatorv1beta1.K
 
 // installKnConsoleCLIDownload creates CR for kn CLI download link
 func (r *ReconcileKnativeServing) installKnConsoleCLIDownload(instance *operatorv1beta1.KnativeServing) error {
-	return consoleclidownload.Apply(instance, r.client, r.scheme)
+	return consoleclidownload.Apply(instance, r.client)
```

Comment:
This file does more than just installing `ConsoleCLIDownload` resources. It also installs Quickstarts, and Dashboard: https://github.com/openshift-knative/serverless-operator/blob/26a310ee58a38e70d3f7bd1d1d057891d06e287f/knative-operator/pkg/controller/knativeserving/knativeserving_controller.go#L320-L333

All of them have no sense without Console, and probably will fail as well.

Reply:
I didnt notice a failure anywhere but can remove them as they depend on the console.